_docs/xcm/integration-overview.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Cross-chain Product Integration
+subtitle: Explain XCM and cross-chain integration work flow with OAK Network
+author: chris
+tags: [xcm]
+---
+
+# Overview
+In this section, we will go over the steps of a cross-chain product integration. For example, the current work undergoing are,
+- LP reward auto-compound with [MangataX DEX](https://app.mangata.finance/). A video demo of the product can be [viewed here](https://drive.google.com/file/d/1Tv8JEEcJszqdE8KcgtXav6lOkq-xlrA0/view?usp=share_link).
+- dApp staking auto-compound for [Shiden Network](https://portal.astar.network/#/astar/dapp-staking/discover)
+
+In both products, Turing Network provides partners with Automation-as-a-service that comes with several technical benefits including
+- Oracle for time and price; Time is intrinsic to the chain and price is retrieved from aggregators such as [DIA](https://app.diadata.org/).
+- Task Storage
+- Task Invocation
+- On/off valve for each user
+
+
+# Integration Workflow
+## 1. Define the product
+### 1.1 Design questions
+Our partner conversation usually starts with a few questions,
+1. Who are the target audience, or beneficial end users of this product?
+    
+    For example, the audience of Shiden <> Turing integration is all users on Shiden Network.
+2. What would be our product offering?
+    
+    Taking Dapp staking as an example, the feature would automate two operations, claim and deposit.
+
+    Another example would be DEX liquidity pool reward compound, which will include three operations, claim, swap, and deposit.
+3. What are the user defined input?
+
+    For auto-compound, normally the UI will need to ask users to provide some preferences, such as,
+    
+    Frequency: how often to compound
+    
+    Amount: percentage of claimed reward to compound(No need since the current re-deposit ratio is always 100%)
```

Comment:
This looked strange when rendered. I think it will look better as a list.
```suggestion
    For auto-compound, normally the UI will need to ask users to provide some preferences, such as:
        - Frequency: how often to compound
        - Amount: percentage of claimed reward to compound (No need since the current re-deposit ratio is always 100%)
```